src/Prompter.ts

```diff
@@ -6,22 +6,26 @@ export interface IPrompter {
   userIO: IUserIO;
   promptList(message: string, choices: (string | Separator)[]): Promise<Answers>;
   promptInput(message: string): Promise<Answers>;
+  promptConfirm(message: string): Promise<Answers>;
+  promptNumber(message: string): Promise<Answers>;
+  promptDate(message: string): Promise<Answers>;
```

Comment:
is `promptDate` asking for an input that is specifically date? if yes, what kind of input `promptInput` is asking for?